src/table/components/__tests__/HeadCellMenu.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { TableContextProvider } from '../../context';
+import { ExtendedTranslator, ExtendedSelectionAPI } from '../../../types';
+import { GeneratedStyling } from '../../types';
+import HeadCellMenu from '../HeadCellMenu';
+
+describe('<HeadCellMenu />', () => {
+  let translator: ExtendedTranslator;
+  let selectionsAPI: ExtendedSelectionAPI;
+  let headerStyle: GeneratedStyling;
+
+  const renderTableHeadCellMenu = (cellCoordMock?: [number, number]) =>
+    render(
+      <TableContextProvider selectionsAPI={selectionsAPI} cellCoordMock={cellCoordMock}>
+        <HeadCellMenu headerStyle={headerStyle} translator={translator} />
+      </TableContextProvider>
+    );
+
+  beforeEach(() => {
+    selectionsAPI = {
+      isModal: () => false,
+    } as ExtendedSelectionAPI;
+    translator = { get: (s) => s } as ExtendedTranslator;
+    headerStyle = {
+      sortLabelColor: '#4287f5',
+      borderStyle: 'solid',
+      borderColor: '#4287f5',
+    };
+  });
+
+  it('should render head cell menu button', () => {
+    const { getByRole } = renderTableHeadCellMenu();
+    expect(getByRole('button').classList.contains('sn-table-head-menu-button')).toBeTruthy();
+    expect(getByRole('button')).not.toBeVisible();
+  });
+
+  it('should open the menu only when the button is clicked', () => {
+    const { getByRole, queryByRole } = renderTableHeadCellMenu();
+
+    expect(queryByRole('menu')).not.toBeInTheDocument();
+    fireEvent.click(getByRole('button'));
+    expect(getByRole('menu')).toBeVisible();
+  });
+
+  it('should show the menu items by opening the menu', () => {
+    const { getByRole, getByText } = renderTableHeadCellMenu();
+
+    fireEvent.click(getByRole('button'));
+    expect(getByText('SNTable.SortItem.SortAscending')).toBeVisible();
+    expect(getByText('SNTable.SortItem.SortDescending')).toBeVisible();
```

Comment:
It's a weird situation, I've already modified the SortItem to MenuItem, but it only passes the test with these strings! 
As I've shown you, the translator doesn't work properly for me.